ateliermecanique-ws/src/main/java/com/champlain/ateliermecaniquews/customerinvoicemanagementsubdomain/presentationlayer/CustomerInvoiceController.java

```diff
@@ -1,4 +1,37 @@
 package com.champlain.ateliermecaniquews.customerinvoicemanagementsubdomain.presentationlayer;
 
+import com.champlain.ateliermecaniquews.appointmentmanagementsubdomain.presentationlayer.AppointmentResponseModel;
+import com.champlain.ateliermecaniquews.authenticationsubdomain.dataLayer.repositories.UserRepository;
+import com.champlain.ateliermecaniquews.customerinvoicemanagementsubdomain.businesslayer.CustomerInvoiceService;
+import com.champlain.ateliermecaniquews.customerinvoicemanagementsubdomain.datalayer.CustomerInvoiceRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1")

```

Comment:
I believe this should be /api/v1/invoices

Reply:
nah brother the class annotation should just  end with v1 (or with v1/test with the new testing path for backend)